### PR TITLE
Use node v6

### PR DESCRIPTION
**Pull request by denar90**
### Proposed changes
 - Updating to node 6
 - Updating all dev dependancies 

Link to the issue: #3199

### CODE REVIEW

**Comment by paulfalgout** [comment]
Also resolves https://github.com/marionettejs/backbone.marionette/issues/3205

**Comment by paulfalgout** [comment]
also just to be thorough..  does the browser testing work?  That's something along with the build that could break but wouldn't be caught by CI.

**Comment by denar90** [comment]
@paulfalgout I tested build in separate branch, looks fine 
https://github.com/marionettejs/backbone.marionette/compare/master...denar90:node-6-build?expand=1
Talking about browser testing, works as did before. But I think some improvements will come with 
https://github.com/marionettejs/backbone.marionette/pull/3198

**Comment by paulfalgout** [comment]
`3,141 additions, 3,036 deletions not shown because the diff is too large. Please use a local Git client to view these changes.` on `/lib/backbone.marionette.js` This is concerning... why is there 100+ line additions?

**Comment by denar90** [comment]
This additional 100+ lines are just comments. I didn't find any proper way to show file diffs, so I made some video https://monosnap.com/file/fC26ZJBSyNLAG7EES7WZYhrVcS159x

**Comment by paulfalgout** [comment]
Ah ok excellent.  I guess older rollup was stripping those out.  This will be nice for annotated source too. 

:+1:

File: package.json [hunk 1]
```diff
@@ -46,44 +46,43 @@
...
+    "jquery": "3.1.0",
+    "jsdom": "9.6.0",
+    "mocha": "3.1.2",
     "opn": "4.0.2",
-    "rollup": "0.34.3",
-    "rollup-plugin-babel": "2.6.1",
-    "rollup-plugin-commonjs": "3.3.1",
-    "rollup-plugin-json": "2.0.1",
+    "rollup": "0.36.3",
+    "rollup-plugin-babel": "2.6.1 ",
```
Comment (paulfalgout) [inline package.json]: ooh just saw this.. extra space for some reason on this line.

**Comment by denar90** [comment]
💃 🎉

**Comment by paulfalgout** [comment]
yeah shoot.. rewatching the video it says `triggerMethod$1`  the build file is odd.  If I clear out the `lib` directory this never builds the minified file either..  _maybe_ the build is ok (but that odd named `triggerMethod` is concerning), but the minified isn't work.

**Comment by paulfalgout** [comment]
ok.. so the build is actually fine.. the issue is we're not building the `min.js`  probably `gulp-filter` or `gulp-sourcemaps` both of those were breaking changes updates

**Pull request merged**